rotkehlchen/tests/utils/premium.py

```diff
@@ -140,15 +136,21 @@ def setup_starting_environment(
         newer_remote_db: bool,
         db_can_sync_setting: bool,
         premium_credentials: PremiumCredentials,
-        remote_data: bytes = REMOTE_DATA,
+        remote_data: Optional[bytes] = None,
         sync_approval: Literal['yes', 'no', 'unknown'] = 'yes',
         sync_database: bool = True,
+        use_remote_data: bool = True,
 ):
     """
     Sets up the starting environment for premium testing when the user
     starts up his node either for the first time or logs in an already
     existing account
     """
+    if remote_data is None and use_remote_data is True:
```

Comment:
Why are you adding a new boolean argument `use_remote_data` here? Can't you simply not use remote data if `remote_data` is `None`? Since you just made it also optional.